tools/system_libs.py

```diff
@@ -940,6 +940,7 @@ def get_files(self):
           'ctime_r.c',
           'timespec_get.c',
           'utime.c',
+          '__map_file.c',
```

Comment:
What does the musl version actually do, in our case?

Reply:
It pretty simply.. it just does open + fstat + mmap + close.. so it should actually work .. assuming mmap works for that file:

https://github.com/emscripten-core/emscripten/blob/15802518cbfc8ea5c62ccf0bdf35500175338f65/system/lib/libc/musl/src/time/__map_file.c#L7-L23

Making it working standlone mode would require a better fake __syscall_fstat64 above.. but I don't think that is a high priority.